pkg/lib/listing.scss

```diff
@@ -631,6 +617,25 @@ tbody tr.listing-ct-noexpand {
     cursor: default;
 }
 
+/* Fix up nav-tabs-pf to work properly */
```

Comment:
Is this something that you would like to report back to PF?

Reply:
I don't know where that comes from. It's in listing.scss, which is the old listing code. And the `nav-tabs-pf` name means it's PF4. And the fact that there's `0px` and a lot of `!important` means the CSS comes from someone else, not me, so I don't know.

I reverted the changes in listing from master, to undo the intertwined changes from this PR and the pre-existing listing widgets. We need a clean break, hence new CSS for the JS+HTML in the code (and this is old CSS).